README.md

```diff
@@ -1,14 +1,96 @@
-# voyage-tasks
 
-Your project's `readme` is as important to success as your code. For 
-this reason you should put as much care into its creation and maintenance
-as you would any other component of the application.
 
-If you are unsure of what should go into the `readme` let this article,
-written by an experienced Chingu, be your starting point - 
-[Keys to a well written README](https://tinyurl.com/yk3wubft).
+<p align="center">
+  <a href="#about-the-project">About</a> •
+  <a href="#roadmap">Roadmap</a> •
+  <a href="#contributing">Contributing</a> •
+  <a href="#feedback">Feedback</a> •
+  <a href="#contributors">Contributors</a> •
+  <a href="#license">License</a> •
+  <a href="#acknowledgements">Acknowledgements</a>
+</p>
 
-And before we go there's "one more thing"! Once you decide what to include
-in your `readme` feel free to replace the text we've provided here.
 
-> Own it & Make it your Own!
+<!-- ABOUT THE PROJECT -->
+
+## About The Project
+
+Website that can monitor and share stock portfolios efficiently with friends.
+
+### Key Features
+- Search Stocks 
+- Create portfolios
+- Share lists with friends that also use the platform.
+
+### Built With
+
+The entire codebase consists in **TypeScript**, here is a list of technologies used:
+
+- **React**: User interface library
+
+### Code Style
+
+- We run Prettier `on-commit`, which means you can write code in whatever style you want and it will be automatically formatted according to the common style when you run `git commit`. We also have ESLint set up, along with Jest tests.
+- We use `commitlint` with `config-conventional`, which means all commit messages should follow conventional rules.
+
+<!-- ROADMAP -->
+
+## Roadmap
+
+See the [open issues](https://github.com/chingu-voyages/v37-bears-team-13/issues) for a list of proposed features (and known issues).
+
+<!-- CONTRIBUTING -->
+
+## Contributing
+
+Any contributions you make are **greatly appreciated**. There are many ways in which you can participate in this project, for example:
+
+- [Submit bugs and feature requests](https://github.com/chingu-voyages/v37-bears-team-13/issues/new/choose), and help us verify as they are checked in
+- Review [source code changes](https://github.com/chingu-voyages/v37-bears-team-13/pulls)
+
+If you are interested in fixing issues and contributing directly to the code base,
+please see the document [How to Contribute](./CONTRIBUTING.md), which covers the following:
```

Comment:
We should either create a `CONTRIBUTING.md` file, or delete this line.

Reply:
You know, I actually thought of deleting that part but I think it would be nice to have a CONTRI... file. But that's just me